apps/deploy-web/src/components/deployments/LeaseRow.tsx

```diff
@@ -66,18 +66,19 @@ export const LeaseRow = React.forwardRef<AcceptRefType, Props>(
       isLoading: isLoadingLeaseStatus
     } = useLeaseStatus(provider, lease, {
       enabled: isLeaseActive && !isServicesAvailable && !!provider?.hostUri && !!localCert,
-      refetchInterval: 10_000,
-      onSuccess: leaseStatus => {
-        if (leaseStatus) {
-          checkIfServicesAreAvailable(leaseStatus);
-        }
-      }
+      refetchInterval: 10_000
     });
+    useEffect(() => {
+      if (leaseStatus) {
+        checkIfServicesAreAvailable(leaseStatus);
+      }
+    }, [leaseStatus]);
+
     const { isLoading: isLoadingProviderStatus, refetch: getProviderStatus } = useProviderStatus(provider, {
       enabled: false,
       retry: false
     });
-    const isLeaseNotFound = error && (error as string).includes && (error as string).includes("lease not found") && isLeaseActive;
+    const isLeaseNotFound = error && (error as unknown as string).includes && (error as unknown as string).includes("lease not found") && isLeaseActive;
```

Comment:
smth like this looks safer. wdut?
```suggestion
    const errorMessage = typeof error === "string" ? error : error?.message;
    const isLeaseNotFound = errorMessage?.includes("lease not found") && isLeaseActive;
```